widget/embedded/src/pages/SelectBlockchainPage.tsx

```diff
@@ -29,6 +33,13 @@ export function SelectBlockchainPage(props: PropTypes) {
   });
   const routeKey = type === 'source' ? 'fromBlockchain' : 'toBlockchain';
 
+  const countActiveCategories = useMemo(
+    () => getCountCategories(blockchains),
+    [blockchains]
+  );
```

Comment:
If you use an object or an array as a dependency in useMemo, it might not work as expected. This is because useMemo uses a shallow comparison to determine whether the dependencies have changed. If you pass an object or an array as a dependency, it will always be considered as changed, even if the contents of the object or array have not changed.

Reply:
Done